packages/abi-coder/test/Interface.test.ts

```diff
@@ -766,4 +766,20 @@ describe('Abi interface', () => {
       );
     });
   });
+
+  describe('decodeLog', () => {
+    it('should return decoded log by id', () => {
+      const data = exhaustiveExamplesInterface.decodeLog('0x01000000000000000000000000000020', '0');
+      expect(data).toEqual({
+        a: true,
+        b: 32,
```

Comment:
The test is failing. The decoded field value is `0`. Probably something is broken.